code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -42,6 +42,17 @@
 	sheet_type = /obj/item/stack/sheet/mineral/bananium
 	canSmoothWith = list(/turf/closed/wall/mineral/bananium, /obj/structure/falsewall/bananium)
 
+/turf/closed/wall/mineral/tranquillite
+	name = "tranquillite wall"
+	desc = "A wall with tranquillite plating. Nearly invisible."
+	icon = 'icons/effects/water.dmi'
+	icon_state = "wet_floor_static"
+	sheet_type = /obj/item/stack/sheet/mineral/tranquillite
+	canSmoothWith = list(/turf/closed/wall/mineral/tranquillite, /obj/structure/falsewall/tranquillite)
+	layer = TURF_LAYER
+	opacity = 0 //actually invisible
```

Comment:
Do you know what opacity does? Genuine question given you put this comment in every `opacity = 0`

Reply:
Opacity=1 blocks light/sight?

Reply:
yeah, if you want it to be TRUE invisible you use `alpha`

Reply:
oh, I thought i also set the alpha to 0 at the same time, I guess its not actually invisible ATM it just has the wet floor sprite.